Cargo.toml

```diff
@@ -24,10 +24,10 @@ colored = "1.8.3"
 probe-rs = { version = "0.6.0" }
 gdb-server = { version = "0.6.0" }
 serde = { version = "1.0", features = ["derive"] }
-config = { version = "0.9.0" }
+config = { version = "0.9.0", features = ["toml", "json", "yaml"], default-features = false }
 probe-rs-rtt = { version = "0.1.0" }
 
 goblin = "0.2.0"
 tui = "0.8.0"
 termion = "1.5.0"
-unicode-width = "0.1.7"
```

Comment:
Hm, this change was not intended (my vim does that automatically). If you prefer missing newlines in files (which I would not agree with :stuck_out_tongue:) you should add an [editorconfig file](https://editorconfig.org/) that a lot of editors will pick up automatically.

Reply:
I don't mind the change. I don't give a fuck about the newline, it is just something that triggers me, which is why my brain makes me remove it automatically. But I agree that it's nicer to have one for the diff. So, good with me.

Reply:
Most editors hide it. The biggest advantage of trailing newlines is that `cat`ing it does not put your command line prompt on the same line as the last line of the output. Also, adding another line won't modify the previous line (adding a newline).

Reply:
Lol i never use cat exceprt for grepping where it doesn't matter :D vim masterrace =D

Reply:
Cat was just an example for line based data processing tools on the command line :wink: POSIX masterrace!